ios/swiftgen.yml

```diff
@@ -0,0 +1,20 @@
+xcassets:
+  - inputs: Sources/Styleguide/Image.xcassets
+    outputs:
+      templatePath: templates/xcassets/swift5.stencil
```

Comment:
I want to know the purpose for this template. If it's okay, let me know.

And also, if you're using SwiftGen, why don't you use for strings as well?

Reply:
In SPM Tool management, using SwiftGen has an issue.

`swift run -c release --package-path Tools swiftgen` : fails
`mint run swiftgen` : success

And I tried to use with `templatePath` instead of `templateName` , it succeeded.
So I use `templatePath` in this PR.

I think this is related to this forum.
https://github.com/apple/swift-evolution/blob/main/proposals/0303-swiftpm-extensible-build-tools.md